qna/DE-certified-online-TSE.md

```diff
@@ -1,8 +1,24 @@
 ## Question
-Is there any certified online TSE available for germany?
+Is there any certified Cloud TSE available for Germany?
+
+## Question
+
+I want to roll out with a Cloud TSE and  I do not want to wait any longer. What options do I have?
+
+## Question
+
+I have ordered a Cloud TSE from my dealer. Can I use an uncertified Cloud TSE in my region until a certified version is available?
 
 ## Metadata tags
 lang-en, market-de, middleware, PosCreator, PosDealer, Consultant
 
 ## Answer
-Currently (18.08.2020), there is **no certified** online TSE available for the german market. 
+Currently (23.09.2020), there is **no certified** Cloud TSE available for the german market. fiskaltrust will offer certified Cloud TSE's in the production environment as soon as they are available on the market. In the meanwhile, we provide uncertified Cloud TSEs for integration and testing in the sandbox environment.
```

Comment:
Please change "we provide uncertified Cloud TSEs" to "we provide uncertified Cloud SignatureCreationUnits" (I know where the SCU ist located).